### PR TITLE
Fix USD cable contact materials

### PR DESCRIPTION
**Pull request by ruziniuuuuu**
## Description

Propagate authored USD physics-material `dynamicFriction` and `restitution` to deformable-cable capsule shapes.

- Use authored dynamic friction as Newton's per-shape sliding friction.
- Preserve builder defaults for unauthored or invalid values.
- Apply one representative material to welded graphs and warn when members differ.

## Validation

- `python -m unittest -v newton.tests.test_import_usd_deformable_cable`: 46 passed.
- `uvx pre-commit run -a`: all hooks passed.
- Downstream GalbotLab: 300 focused VBD/cable/coupling/MJCF tests passed; FoldCloth 1/1; InsertUsb 2/2 with Newton and 2/2 with OVRTX.

## Summary by CodeRabbit

* **Bug Fixes**
  * USD cable capsules now honor authored contact friction and restitution.
  * Missing contact values preserve the existing defaults; invalid values (e.g., negative/NaN or out-of-range restitution) are ignored with warnings.
  * Welded cable components now apply consistent contact properties across generated shapes.
* **Tests**
  * Added coverage for non-welded and welded deformable cable contact-material propagation and edge cases.
* **Chores**
  * Addressed compiler warnings related to overflowing integer constants during kernel compilation.

### CODE REVIEW

**Comment by ruziniuuuuu** [comment]
Updated the branch onto upstream main bfe8c313 and addressed the review finding. Welded cable heterogeneity now includes sanitized friction and restitution in the representative-material signature and warns when members differ. Added coverage for differing welded contact properties, unauthored default preservation, negative and non-finite friction, and out-of-range restitution. Validation: all 46 deformable cable importer tests pass; uvx pre-commit run -a passes; downstream GalbotLab FoldCloth succeeds physically and the OVRTX USB cable path completes a full insertion.

**Comment by ruziniuuuuu** [comment]
Rebased the patch head onto current `main` (`bd449510`); the updated immutable head is `5fc6d961`. The cable-material diff is unchanged.

Validation:
- `test_import_usd_deformable_cable.py`: 46 passed + 29 subtests
- GalbotLab Newton full-DR FoldCloth: 1/1
- GalbotLab OVRTX full-DR InsertUsb: 2/2, plus 4 parallel rows × 3 cameras at 4/4
- GalbotLab suite: 743 passed, 41 skipped

File: CHANGELOG.md [hunk 1]
```diff
@@ -104,6 +104,8 @@
 - Report malformed MJCF free-joint and inertial inputs with deterministic validation errors, and ignore MJCF mesh geom `size` lengths consistently.
 - Fix MJCF imports ignoring material and inline RGBA colors on primitive geoms.
 - Fix Style3D solver divergence caused by isolated vertices.
+- Fix USD cable capsules ignoring authored contact friction and restitution.
+- Fix compiler warnings about overflowing int32 constants when compiling SDF texture and `SensorTiledCamera` render kernels.
```
Comment (eric-heiden) [inline CHANGELOG.md]: This changelog entry is duplicated, please fix.
